pycondor/dagman.py

```diff
@@ -289,11 +333,31 @@ def build(self, makedirs=True, fancyname=True):
         self.submit_name = name
         checkdir(self.submit_file, makedirs)
 
+        if single_file:
+            # in single file mode, write the single job submit file
+            job_submit_file = os.path.join(self.submit, '{}_job.submit'.format(name))
+            self.job_submit_file = job_submit_file
+            checkdir(self.job_submit_file, makedirs)
+
+            if not os.path.exists(job_submit_file):
+                # Write lines to the single job submit file
+                with open(job_submit_file, 'w') as file:
```

Comment:
This is a minor comment, but can you replace `file` here with something else, perhaps `f` or `submit_file`, to avoid confusion with the built-in `file` function in Python 2